### DISCUSSION TITLE
High LCP render delay on web mobile view

### DISCUSSION THREAD

**Discussion by Blendfoul** [tamagui/tamagui#2820] [General]
### Current Behavior

The current implementation of fonts on the website is adversely affecting the LCP metric, specifically in the mobile view. This problem does not appear in the desktop view. 

Could there be any issue on how the mobile font loading strategy is being handled by the configuration that is causing this issue?

The issue is also reproducible on the [Tamgui](https://tamagui.dev) site, gonna leave the lighthouse report here for context:
[Lighthouse Result](https://pagespeed.web.dev/analysis/https-tamagui-dev/n9gsj8dbg3?form_factor=mobile)

### Expected Behavior

Font loading should not have a huge render delay on mobile view

### Tamagui Version

```markdown
1.101.7
```

### Platform (Web, iOS, Android)

```markdown
Web
```

### Reproduction

```markdown
Serve a production build of a nextjs app with a tamagui ui implementation
```

**Comment by natew** [comment]
We released it on a new stack recently, not sure if this was posted before or after that, but I did test to be sure fonts loaded as soon as possible. It's definitely possible things regressed though.

**Comment by natew** [comment]
Going to move to discussion as its not a core issue.
